cpp/src/shim/cufile.cpp

```diff
@@ -27,17 +27,7 @@ namespace kvikio {
 #ifdef KVIKIO_CUFILE_FOUND
 cuFileAPI::cuFileAPI()
 {
-  // CUDA versions before CUDA 11.7.1 did not ship libcufile.so.0, so this is
-  // a workaround that adds support for all prior versions of libcufile.
-  void* lib = load_library({"libcufile.so.0",
-                            "libcufile.so.1.3.0" /* 11.7.0 */,
-                            "libcufile.so.1.2.1" /* 11.6.2, 11.6.1 */,
-                            "libcufile.so.1.2.0" /* 11.6.0 */,
-                            "libcufile.so.1.1.1" /* 11.5.1 */,
-                            "libcufile.so.1.1.0" /* 11.5.0 */,
-                            "libcufile.so.1.0.2" /* 11.4.4, 11.4.3, 11.4.2 */,
-                            "libcufile.so.1.0.1" /* 11.4.1 */,
-                            "libcufile.so.1.0.0" /* 11.4.0 */});
+  void* lib = load_library("libcufile.so.0");
```

Comment:
Since we are using this with only one library, think we could drop this as well

https://github.com/rapidsai/kvikio/blob/54c420a652bf29e3c7de9ac2e2e19af07de7c256/cpp/src/shim/utils.cpp#L36-L48

Though perhaps this functionality is useful in the future

Reply:
Handled in commit: https://github.com/rapidsai/kvikio/pull/770/commits/999042a487671dd6bc771f6b049703917026ac13